stacks-common/src/libcommon.rs

```diff
@@ -62,5 +62,5 @@ pub mod consts {
 
     /// The number of StackerDB slots each signing key needs
     ///  to use to participate in DKG and block validation signing.
-    pub const SIGNER_SLOTS_PER_USER: u32 = 12;
+    pub const SIGNER_SLOTS_PER_USER: u32 = 13;
```

Comment:
Is this required anymore? Can we use MessageSlotID::ALL? Or would that cause a circular reference here?

Reply:
Yeah, that would cause a circular reference.